dhis-2/dhis-api/src/main/java/org/hisp/dhis/deduplication/MergeObject.java

```diff
@@ -43,4 +44,18 @@
     private List<String> trackedEntityAttributes;
```

Comment:
What if we initialise the collections here?
If it works don't need of the init method

Reply:
Done

Reply:
This should work, along with `@Builder.Default` on fields and `@JsonDeserialize( builder = MergeObject.MergeObjectBuilder.class)` on merge object